test/test_other.py

```diff
@@ -15807,6 +15807,9 @@ def test_rollup(self):
   def test_rlimit(self):
     self.do_other_test('test_rlimit.c', emcc_args=['-O1'])
 
+  def test_stub_pthread_getattr_np(self):
+    self.do_other_test('test_stub_pthread_getattr_np.c')
```

Comment:
Actually can this simply be combined with the existing `test_pthread_stub` in this fvile?  (Or at least co-located with it).